Assets/StreamingAssets/Quests/A0C00Y16.txt

```diff
@@ -170,36 +164,16 @@ Message:  1054
 <ce>                      you made that Smiley suffer,
 <ce>                         %pcn. This is a day of
 <ce>                          great sorrow for me.
- 
-
--- Symbols used in the QRC file:
---
---                %g occurs 20 times.
---               %g2 occurs 17 times.
---               %g3 occurs 1 time.
---              %oth occurs 2 times.
---              %pcf occurs 1 time.
---              %pcn occurs 4 times.
---              %qdt occurs 1 time.
---               %ra occurs 5 times.
---  __missingperson_ occurs 1 time.
---         _friend1_ occurs 5 times.
---         _friend2_ occurs 3 times.
---         _friend3_ occurs 3 times.
---         _friend4_ occurs 1 time.
---            _gold_ occurs 7 times.
---   _missingperson_ occurs 26 times.
---      _questgiver_ occurs 10 times.
 
 QBN:
 Item _gold_ gold
 
 Person _questgiver_ group Questor
-Person _missingperson_ face 1 group Group_6.0 local
-Person _friend1_ group Local_3.1 local
-Person _friend2_ group Local_3.1 local
-Person _friend3_ group Local_3.1 local
-Person _friend4_ group Local_4.1 local
+Person _missingperson_ face 1 group Group_6.0
```

Comment:
@df-ferital Do you think it's right to remove these flags? From what I understand, the classic quest has the "local" flag, and if it's removed, it's going to be 50% chance of either "local" or "remote", right?

Seems like folks are still confused as to what to do with these flags, especially since the VS Code plugin still doesn't recognize it

Reply:
In case of confusion, I talked with JayH on Discord and he reverted those changes. I still hope you can clarify in case I got something wrong

Reply:
I don't agree with this change. There is no need to set the location scope of the questgiver since he/she is always local, since we talk to him to get the quest. However, missingperson should remain local, else he/she can be placed in another town, which is not what we want there.